### PR TITLE
Add secrets integration to jobsrv/worker

### PR DESCRIPTION
**Pull request by elliott-davis**
Signed-off-by: Elliott Davis <elliott@excellent.io>

### CODE REVIEW

**Comment by thesentinels** [comment]
Thanks for the pull request! Here is what will happen next:
 1. Your PR will be reviewed by the maintainers
 2. If everything looks good, one of them will `approve` it, and your PR will be merged.

Thank you for contributing!

File: components/builder-jobsrv/src/server/worker_manager.rs [hunk 1]
```diff
@@ -493,6 +501,105 @@ impl WorkerMgr {
...
+        let mut origin_req = OriginGet::new();
+        let origin_name = job.get_project().get_origin_name().to_string();
+        let origin_owner_id = job.get_project().get_owner_id();
+        origin_req.set_name(origin_name.clone());
+        match self.route_conn.route::<OriginGet, Origin>(&origin_req) {
+            Ok(origin) => secrets_request.set_origin_id(origin.get_id()),
+            Err(e) => return Err(Error::NetError(e)),
+        };
+
+
```
Comment (chefsalim) [inline components/builder-jobsrv/src/server/worker_manager.rs]: Minor - remove extra whitespace line

File: components/builder-worker/src/runner/studio.rs [hunk 2]
```diff
@@ -97,6 +97,16 @@ impl<'a> Studio<'a> {
         cmd.env("PATH", env::var("PATH").unwrap_or(String::from(""))); // Sets `$PATH`
         cmd.env(NONINTERACTIVE_ENVVAR, "true"); // Disables progress bars
         cmd.env("TERM", "xterm-256color"); // Emits ANSI color codes
+        for secret in self.workspace.job.get_secrets() {
+            cmd.env(
+                format!(
+                    "HAB_STUDIO_SECRET_{}",
+                    secret.get_decrypted_secret().get_name()
+                ),
+                secret.get_decrypted_secret().get_name(),
```
Comment (chefsalim) [inline components/builder-worker/src/runner/studio.rs]: Should this be get_value() ?

**Comment by elliott-davis** [comment]
@thesentinels approve

**Comment by thesentinels** [comment]
:metal: I am testing your branch against master before merging it. We do this to ensure that the master branch is never failing tests.

**Comment by thesentinels** [comment]
:neckbeard: Travis CI has [started testing this PR](https://travis-ci.org/habitat-sh/builder/builds/353537057).

**Comment by thesentinels** [comment]
:sparkling_heart: Travis CI [reports this PR passed](https://travis-ci.org/habitat-sh/builder/builds/353537057).

It always makes me feel nice when humans approve of one anothers work. I'm merging this PR now.

I just want you and the contributor to answer me one question:

gif-keyboard-3280869874741411265

**Pull request merged**